### ISSUE TITLE
ICS ?: Relayer Query Endpoints

### ISSUE THREAD

**Issue by mossid** [cosmos/ibc#61]
The chains need to expose endpoints to query packets, headers, evidence, etc., in a standard form for the relayers to connect easily without acknowledging about the internal consensus logic. The relayers should be able to connect to any node which is running a full node of an IBC chain and start relaying packets to another.

**Comment by cwgoes** [comment]
I agree but I am not sure if exact REST formats should be in scope of this repo, that sounds like an implementation spec. We should definitely define the exposed data and proof requirements.

**Comment by cwgoes** [comment]
What I think we should do is define `pendingDatagrams` clearly - see https://github.com/cosmos/ics/pull/37#issuecomment-478254700

**Comment by cwgoes** [comment]
Closing in favor of https://github.com/cosmos/ics/issues/217.

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
